api_server.py

```
@@ -150,19 +171,25 @@ def total_hrs(p):
             "ba_non_ttr": 0,
         }
-    with get_db() as conn:
-        placeholders = ",".join("?" * len(available_ids))
-        rows = conn.execute(
-            f"SELECT role, skills FROM crew WHERE id IN ({placeholders})", available_ids
-        ).fetchall()
+    else:
+        with get_db() as conn:
+            placeholders = ",".join("?" * len(available_ids))
+            rows = conn.execute(
+                f"SELECT role, skills FROM crew WHERE id IN ({placeholders})",
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)